Add dere to card full desc

TheFenix2000 opened the pull request:
It was only present in s.card-, why? I think it should be in both if it's card's full stats (almost), so..

TheFenix2000 committed:
Add power to card full desc

Added dere to s.card.
Why was it only in `s.card-` and other stats were in both?

TheFenix2000 committed:
Update Sanakan.csproj

Up VERSION

MZKNEK commented:
Dere can be seen in the picture of the card, and it's officially called `charakter` in game.

TheFenix2000 committed:
change name

So could't it be like this then? Not everyone knows all the crystals and in my opinion it would be better to have it's visual and wordy representation.
